content/en/docs/k8s-operator/_index.md

```diff
@@ -0,0 +1,68 @@
+---
+title: OpenTelemetry Operator for Kubernetes
+linkTitle: K8s Operator
+weight: 11
+description:
+  An implementation of a Kubernetes Operator, that
+  manages collectors and auto-instrumentation of the workload using OpenTelemetry
+  instrumentation libraries.
+spelling: cSpell:ignore Otel
+aliases: [/docs/operator]
+---
+
+## Introduction
+
+The OpenTelemetry Operator is an implementation of a [Kubernetes Operator](https://coreos.com/operators/).
+
+The operator manages:
+
+- OpenTelemetry Collector
+- auto-instrumentation of the workloads using OpenTelemetry instrumentation libraries
```

Comment:
This can be tackled independently, but the fact that it's mentioned and there isn't an example of how to inject autoinstrumentation into a cluster (pod? I'm not a k8s expert 😅) will definitely be confusing. We should link to the existing sections in the repo that do show this.

Reply:
Note that this is already mentioned here 

https://opentelemetry.io/docs/instrumentation/

> If you are using Kubernetes, you can use the [OpenTelemetry Operator for Kubernetes](https://github.com/open-telemetry/opentelemetry-operator) to [inject auto-instrumentation libraries](https://github.com/open-telemetry/opentelemetry-operator#opentelemetry-auto-instrumentation-injection) for Java, Node.js and Python into your application.

Reply:
I am going to link to the same place as we have it with /docs/instrumentation -- later on we should consider having a proper end-to-end example in the docs (some kind of "getting started" similar to the languages, maybe even reusing them?).

The goal of this PR is to have something from which we can start, so let's collect all those points and then add them in one by one :-)